lib/chef/audit/runner.rb

```diff
@@ -45,6 +45,10 @@ def num_total
         RSpec.world.reporter.examples.size
       end
 
+      def exclusion_pattern exclusion_path
+          Regexp.new(exclusion_path.split(/[\\\/]/)[0..2].join("[\\\/]"))
```

Comment:
Why switch to `\\\/` instead of `File::SEPARATOR`?  Will this match on systems like AIX and solaris (that may be a dumb question, I don't know)?  What about windows filesystems where the install path contains an escaped space?